aiven/kafka_topic_availability.go

```diff
@@ -19,6 +20,16 @@ type KafkaTopicAvailabilityWaiter struct {
 	TopicName   string
 }
 
+var renewing int32
+
+func isRenewing() bool {
```

Comment:
isRenewing / finishRenewing as names do not pair that well, also isXX() kind of only implies a check not state change, so I would recommend changing isRenewing to something that makes it more obvious that on success the state has changed

Reply:
Indeed bad names, I did struggle to find an appropriate one and decided to use semaphores instead to make code a bit more understandable.